src/microbe_stage/PatchMapGenerator.cs

```diff
@@ -583,7 +592,7 @@ private static void ConnectPatchesBetweenRegions(PatchRegion region, Random rand
         {
             case PatchRegion.RegionType.Ocean or PatchRegion.RegionType.Sea:
             {
-                foreach (var adjacent in region.Adjacent)
+                foreach (var adjacent in region.Adjacent.CloneShallow().Keys)
```

Comment:
I assume this is here as the adjacent data is now modified here? If so the usual pattern to use is `region.Adjacent.Keys.ToList()` to take a copy of the keys.